<ui version="4.0" >
 <class>toConnectionImport</class>
 <widget class="QDialog" name="toConnectionImport" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import Connections</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;From:</string>
     </property>
     <property name="buddy" >
      <cstring>toolComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="toolComboBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="runButton" >
     <property name="text" >
      <string>&amp;Run...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="notificationLabel" />
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QTableView" name="tableView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>toConnectionImport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>toConnectionImport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
